newIDE/app/src/EventsSheet/EventsTree/Renderers/CommentEvent.js

```diff
@@ -1,83 +1,79 @@
 // @flow
+import { t } from '@lingui/macro';
+
 import * as React from 'react';
-import ReactDOM from 'react-dom';
 import classNames from 'classnames';
-import { type EventRendererProps } from './EventRenderer';
-import { rgbToHex } from '../../../Utils/ColorTransformer';
+import TextField from '../../../UI/TextField';
 import {
   largeSelectedArea,
   largeSelectableArea,
   selectableArea,
+  disabledText,
 } from '../ClassNames';
+import { type EventRendererProps } from './EventRenderer';
 const gd = global.gd;
 
 const styles = {
   container: {
-    minHeight: 30,
     display: 'flex',
-    backgroundColor: '#fbf3d9',
-  },
-  text: {
-    flex: 1,
-    whiteSpace: 'pre-line',
-    margin: 0,
+    flexWrap: 'wrap',
     padding: 5,
+    overflow: 'hidden',
+    backgroundColor: '#fbf3d9',
   },
-  textArea: {
-    padding: 5,
-    flex: 1,
-    boxSizing: 'border-box',
+  textField: {
     width: '100%',
-    fontSize: 14,
+    fontSize: 18,
```

Comment:
I think comments are too big. I kind of liked the fact that they are smaller than groups, because groups are like a Header in a text, while comment is text:

## This is a group 

This is a comment blablabla